api/CustomCommand.md

```diff
@@ -8,14 +8,15 @@
 
 `custom object node func_name [params]`
 
-Calls the given Godot function on a (child) node of a registered `ESCitem`.
+Executes the specified Godot function. This function must be in a script
+attached to a child node of an `ESCitem`.
```

Comment:
I think it's worth keeping that the `ESCItem` must be registered, no, @dploeger ?

Reply:
Happy to put it back in. I couldn't spot anything in the documentation about registering `ESCitem`s so took "registered" to mean that it meant any valid ESCitem. Searching the documentation for "register[ed]", I can see a few mentions of things being registered, but nothing that tells me what registration is for, what needs to be registered, how you register something (other than "register_object"), what happens if you don't register it, etc.

Reply:
This PR is closed, but for the sake of completeness @BHSDuncan, FYI I've readded "registered" to the documentation line, as well as raising a new issue calling out that "registration" isn't documented anywhere.